sdk/storage/Azure.Storage.Files.Shares/tests/ShareClientTestFixtureAttribute.cs

```diff
@@ -29,7 +29,7 @@ public ShareClientTestFixtureAttribute()
                 StorageVersionExtensions.MaxVersion)
         {
             RecordingServiceVersion = StorageVersionExtensions.MaxVersion;
-            LiveServiceVersions = new object[] { StorageVersionExtensions.LatestVersion, };
+            LiveServiceVersions = new object[] { StorageVersionExtensions.MaxVersion, };
```

Comment:
Is this just a name change? or is there a difference between latest and max version

Reply:
woops. I didn't see your other comment